doc/documentation/custom_viz.rst

```diff
@@ -12,7 +12,17 @@ have to delve into the underlying engine to get a different look.
 Changing the display background
 -------------------------------
 
-The display background can take any valid matplotlib color.
+The display background can take any valid matplotlib color (i.e.,
+it can be a tuple of rgb values, an rgb hex string, or a named HTML
+color.
+
+Changing the display size
+-------------------------
+
+The default display window is 800px by 800px, but this is also
+customizable. You can set the size of a square window with the ``size``
+parameter or make a rectangular window with the ``width`` and ``height``
```

Comment:
Ahh, right. So maybe deprecate `width` and `height` and allow `size` to be an int or tuple/list of two int?